diff -rub deps.old/ccc/codechicken/core/launch/CodeChickenCorePlugin.java deps/ccc/codechicken/core/launch/CodeChickenCorePlugin.java
--- deps.old/ccc/codechicken/core/launch/CodeChickenCorePlugin.java	Wed Nov 27 16:58:09 2013
+++ deps/ccc/codechicken/core/launch/CodeChickenCorePlugin.java	Wed Nov 27 16:59:11 2013
@@ -52,12 +52,6 @@
         MCPDeobfuscationTransformer.load();
     }
     
-    @Override
-    public String[] getLibraryRequestClass()
-    {
-        return null;
-    }
-
     public static void versionCheck(String reqVersion, String mod)
     {
         String mcVersion = (String) FMLInjectionData.data()[4];
diff -rub deps.old/ccc/codechicken/core/launch/DepLoader.java deps/ccc/codechicken/core/launch/DepLoader.java
--- deps.old/ccc/codechicken/core/launch/DepLoader.java	Wed Nov 27 16:58:09 2013
+++ deps/ccc/codechicken/core/launch/DepLoader.java	Wed Nov 27 16:59:11 2013
@@ -675,10 +675,4 @@
         
         return null;
     }
-    
-    @Override
-    public String[] getLibraryRequestClass()
-    {
-        return null;
-    }
 }
diff -rub deps.old/ccl/codechicken/lib/packet/MetaPacket.java deps/ccl/codechicken/lib/packet/MetaPacket.java
--- deps.old/ccl/codechicken/lib/packet/MetaPacket.java	Wed Nov 27 16:58:11 2013
+++ deps/ccl/codechicken/lib/packet/MetaPacket.java	Wed Nov 27 16:59:29 2013
@@ -58,13 +58,17 @@
     }
 
     @Override
-    public void writePacketData(DataOutput dataout) throws IOException
+    public void writePacketData(DataOutput dataout)
     {
         //send a dummy 250
         super.writePacketData(dataout);
         
+        try {
         for(Packet p : packets)
             Packet.writePacket(p, dataout);
+        } catch (IOException e) {
+            throw new RuntimeException(e);
+        }
         
         //adjust sent size for Packet.writePacket
         Packet.sentSize-=getPacketSize()-super.getPacketSize();
diff -rub deps.old/nei/codechicken/nei/asm/NEICorePlugin.java deps/nei/codechicken/nei/asm/NEICorePlugin.java
--- deps.old/nei/codechicken/nei/asm/NEICorePlugin.java	Wed Nov 27 16:58:14 2013
+++ deps/nei/codechicken/nei/asm/NEICorePlugin.java	Wed Nov 27 16:59:11 2013
@@ -12,12 +12,6 @@
 public class NEICorePlugin implements IFMLLoadingPlugin, IFMLCallHook
 {
     @Override
-    public String[] getLibraryRequestClass()
-    {
-        return null;
-    }
-
-    @Override
     public String[] getASMTransformerClass()
     {
         CodeChickenCorePlugin.versionCheck(CodeChickenCorePlugin.mcVersion, "NotEnoughItems");
